doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,146 @@
 
 This tutorial has following sections:
 
-- [Prerequisites](#prerequisites)
-- [Setup your repo](#setup-your-repo)
-- [Getting started](#getting-started)
-  - [Create the project](#create-the-project)
-  - [Create the generated layer](#create-the-generated-layer)
-- [Tests](#tests)
-- [Samples](#samples)
-- [Snippets](#snippets)
-- [Export Public API](#export-public-api)
-- [README](#readme)
-- [Changelog](#changelog)
+- [Azure SDK Code Geneneration Quickstart Tutorial (Data Plane)](#azure-sdk-code-geneneration-quickstart-tutorial-data-plane)
+  - [Prerequisites](#prerequisites)
+  - [Setup your repo](#setup-your-repo)
+  - [Getting started - shippable package](#getting-started---shippable-package)
+    - [Create the project folder](#create-the-project-folder)
+      - [1. install dotnet tempate](#1-install-dotnet-tempate)
+      - [2. dotnet new project](#2-dotnet-new-project)
+      - [3. update the solution file if needed](#3-update-the-solution-file-if-needed)
+    - [Generate client library](#generate-client-library)
+    - [Export Public API](#export-public-api)
+  - [Improve the SDK library package](#improve-the-sdk-library-package)
+    - [Tests](#tests)
+    - [Samples](#samples)
+    - [Snippets](#snippets)
+    - [README.md](#readmemd)
+    - [Changelog](#changelog)
 
 <!-- /TOC -->
 
 ## Prerequisites
 
-- Install VS 2019 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
+- Install VS 2020 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
   - Need at least .NET Framework 4.6.1 and 4.7 development tools
-- Install the **.NET Core cross-platform development** workloads in VisualStudio
-- Install **.NET Core 5.0.301 SDK** for your specific platform. (or a higher version within the 5.0.*** band)  (https://dotnet.microsoft.com/download/dotnet-core/5.0)
+  - Install the **.NET Core cross-platform development** workloads in VisualStudio
+- Install **.NET 6.0 SDK** for your specific platform. (or a higher version)
+- Install **.NET core 3.1** for your specific platform.
 - Install the latest version of [git](https://git-scm.com/downloads)
-- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 6 or higher.
+- Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 7 or higher.
 - Install [NodeJS](https://nodejs.org/) (14.x.x).
 
 ## Setup your repo
 
 - Fork and clone an [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repo. Follow the instructions in the [.NET Contributing.md](https://github.com/Azure/azure-sdk-for-net/issues/12903) to fork and clone the `azure-sdk-for-net` repo.
 - Create a branch to work in. 
 
-## Getting started
+## Getting started - shippable package
```

Comment:
Could you help clarify the goal of adding this to this section header?  Are we saying that once the reader has completed the Getting Started section, they will have a shippable package?  If so, we should probably not call it getting started, maybe "Create a shippable package" (but I'd like us to clarify what we mean by shippable as noted elsewhere).

Reply:
Resolved per above.